Cargo.toml

```diff
@@ -78,7 +78,7 @@ fluvio_ws_stream_wasm = "0.7.0"
 bytes = "1.0.0"
 lazy_static = "1.2.0"
 num_cpus = "1.10.1"
-futures-util = { version = "0.3.5", features = ["sink"] }
+futures-util = { version = "0.3.5", features = ["sink", "io"] }
```

Comment:
do we need io everywhere or can be scoped?  probably same for `sink` as well

Reply:
Huh. The io flag was the one that was actually needed. I made it only used for the tls features.